src/utils/work/getExtraDependencies.js

```diff
@@ -17,12 +17,17 @@ const getClusteringSettings = async (experimentId, body, dispatch, getState) =>
   return clusteringSettings;
 };
 
-// Check that the cell sets within the selected cellSetKey didn't change
+// Check that the cell sets within the selected selectedCellSet didn't change
 // e.g., if cell set was deleted we can't use cache
 const getSelectedCellSet = async (experimentId, body, dispatch, getState) => {
+  if (!body.downsampleSettings) return '';
```

Comment:
Why do we return `''` if there are no `downsampleSettings` in the body?

Reply:
because if we are not performing downsampling, these dependencies don't exist. Perhaps I should add a comment explaning that

Reply:
Added a comment

Reply:
pedantic thing, but the comment for `getSelectedCellSet` should be `// If not downsampling, then there's no dependency on **selected cell sets**`, right? At the moment it says `groupedTracks`.

Reply:
Updated this comment but due to merging the 2 downasmple dependency getters the comment was changed further